.github/workflows/ci.yml

```
@@ -1,71 +1,19 @@
 name: CI
-on: [push, pull_request]
-env:
-  RAILS_ENV: test
-  DATABASE_URL: postgresql://postgres:@localhost/test
-  DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
-  BUNDLE_WITHOUT: "journald:development:console:mysql2:sqlite"
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
 jobs:
   rubocop:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Setup Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: 2.7
-          bundler-cache: true
-      - name: Run rubocop
-        run: bundle exec rubocop
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
```

You can either declare the task in `Rakefile`, or override the command now:
```suggestion
    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
    with:
      command: bundle exec rubocop --parallel --format github
```

---

Sorry, wrong repo. This isn't an issue here.